test/framework/e2e_framework.go

```diff
@@ -45,14 +45,20 @@ type E2eFramework struct {
 var (
 	nodeToolStatusUN = regexp.MustCompile("UN\\s\\s")
 	nodeToolStatusDN = regexp.MustCompile("DN\\s\\s")
+	CqlshBin         = "/opt/cassandra/bin/cqlsh"
```

Comment:
I think we can avoid to export this variable. Also. I'd rather see it as a private (unexported) field of the `E2eFramework` struct, rather than a global variable.

Reply:
Sure thing